docs/Developers-Guide.md

```diff
@@ -153,76 +170,29 @@ opinions, foster discussion, and move toward consensus. When responding to a pro
 
-Of course you may always fork the [OTP repo on GitHub](https://github.com/opentripplanner/OpenTripPlanner/) 
-and submit your changes as a pull request, or develop and share whatever features you like on your fork even if they
-are not included in mainline OTP.
+Of course you may always fork the [OTP repo on GitHub]
+(https://github.com/opentripplanner/OpenTripPlanner/) and submit your changes as a pull request, or
+develop and share whatever features you like on your fork even if theyare not included in mainline 
+OTP.
 
 
 ## Code style
 
-### Java
-
-OpenTripPlanner uses the same code formatting and style as the [GeoTools](http://www.geotools.org/) and 
-[GeoServer](http://geoserver.org) projects. It's a minor variant of the
-[Sun coding convention](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf). Notably, **we do not use tabs** 
-for indentation and we allow for lines up to 100 characters wide.
-
-The Eclipse formatter configuration supplied by the GeoTools project allows comments up to 150 characters wide.
-A modified version included in the OpenTripPlanner repository will wrap comments to the same width as lines of code, 
-which makes for easier reading in narrow windows (e.g. when several documents are open side-by-side on a wide display).
-
-If you use Eclipse, you should do the following to make sure your code is automatically formatted correctly:
-
-1. Open the project `Properties` (right-click on the project directory in Eclipse and select `Properties` or choose `Project` -> `Properties`).
-
-2. Select `Java`, then `Code Style`, and finally `Formatter`.  
-
-3. Check the `Enable project specific settings` checkbox.
-
-4. Click `Import...`, select the `formatter.xml` file in the root of the OpenTripPlanner git repository, and click `Open`.
-
-5. Click `OK` to close the `Properties` window.
-
-
-### JavaScript
-
-As of #206, we follow [Crockford's JavaScript code conventions](http://javascript.crockford.com/code.html). Further guidelines include:
-
- * All .js source files should contain one class only
-
- * Capitalize the class name, as well as the source file name (a la Java)
-
- * Include the GNU LGPL header at top of file, i.e., `/* This program is free software:...*/`
-
- * Include the namespace definition in each and every file: `otp.namespace("otp.configure");`
-
- * Include a class comment. For example,                                                                                                      
-
-```java
-    /**
-     * Configure Class
-     *
-     * Purpose is to allow a generic configuration object to be read via AJAX/JSON, and inserted into an Ext Store
-     * The implementation is TriMet route map specific...but replacing ConfigureStore object (or member variables) with
-     * another implementation, will give this widget flexibility for other uses beyond the iMap.
-     *
-     * @class
-     */
-```
+[The OTP styleguid is described in its own styleguide.](developersguide/Codestyle.md)
```

Comment:
Suggest replacing with: `The OTP code style is described on a separate [style guide page](developersguide/Codestyle.md)`.

Reply:
Replaced.